File: Commercial/Go_Commercial_Controller/Building.go
package main

// Building type
type Building struct {
	Floors, Basements int
}

// NewBuilding : Building factory function
func NewBuilding(floors int, basements int) Building {
	b := Building{
		Floors:    floors,
		Basements: basements,
	}
	return b
}

File: Commercial/CSharp_Commercial_Controller/Building.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace Commercial_Controller
{
    public class Building
    {
        public int Floors;
        public int Basements;

        public Building(int Floors, int Basements)
        {
            this.Floors = Floors;
            this.Basements = Basements;
        }
    }
}
